xchange-binance/src/main/java/org/knowm/xchange/binance/dto/trade/BinanceOrderStatus.java

```diff
@@ -12,9 +12,9 @@
   EXPIRED;
 
   @JsonCreator
-  public static OrderStatus getOrderStatus(String s) {
+  public static BinanceOrderStatus getOrderStatus(String s) {
```

Comment:
doesn't Jackson has a similar behaviour by default for enums if you do not suggest a @JsonCreator in them?
this seams unnecessary in all enums

Reply:
Do you mean revert specific naming for binance order status? Does it seem useful for coding?

Reply:
I mean that if we remove this method, Jackson should use BinanceOrderStatus.valueOf by it self